lib/rpcServer/commands/getBlockHeaders.js

```diff
@@ -12,14 +12,15 @@ const getBlockHeadersFactory = (coreAPI) => {
    * Returns block headers
    * @typedef getBlockHeaders
    * @param args - command arguments
-   * @param {number} args.offset - block height starting point
-   * @param {number} args.limit - number of block headers to return
+   * @param {number|string}  args.offset - block height/hash starting point
+   * @param {number} [args.limit=1] - number of block headers to return
+   * @param {boolean} [args.verbose=false] - set true to get headers as deserialized json
    * @return {Promise<Array<Object>>}
    */
   async function getBlockHeaders(args) {
     validator.validate(args);
-    const { offset, limit } = args;
-    return coreAPI.getBlockHeaders(typeof offset === 'number' ? await coreAPI.getBlockHash(offset) : offset, limit);
+    const { offset, limit, verbose } = args;
+    return coreAPI.getBlockHeaders(typeof offset === 'number' ? await coreAPI.getBlockHash(offset) : offset, limit, verbose);
```

Comment:
Could we decompose it somehow?

Reply:
> Could we decompose into separate lines/vars?

Sure. Is it ok now?

Reply:
I would store the first argument to a separate var just to increase readablity